### DISCUSSION TITLE
sendMessage fetch

### DISCUSSION THREAD

**Discussion by Jexly0x** [telegraf/telegraf#1631] [Q&A] [answered]
How do I indent a message in a fetch request? For clarity, I will attach two screen-shots

![Screenshot_1](https://user-images.githubusercontent.com/97974591/171014499-316cb8b5-a6dd-490f-bbc3-c2d7f0b71fb7.png)
![code](https://user-images.githubusercontent.com/97974591/171014511-aca2388e-01a1-4c30-8bda-68e58dc2c2a6.png)

**Comment by Jexly0x** [comment, marked as answer]
I found the answer. Too bad no one helped me. Still, if someone has an answer on how to make a few indents, I will be glad!

**Reply by Gauntletww** [reply]
Bro of use axios it's quite easy then fetch try else i will help you too

**Comment by MKRhere** [comment]
You must URL-encode whitespaces in URLs. Of course if you actually used Telegraf, this will automatically be done for you.

https://en.wikipedia.org/wiki/Percent-encoding
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent
